src/modules/CourseSearch/components/RecommendationText/index.tsx

```diff
@@ -0,0 +1,133 @@
+import { gql, useQuery } from '@apollo/client'
+import styled from '@emotion/styled'
+import { Link, Typography } from '@mui/material'
+import { Box } from '@mui/system'
+import useGoogleOptimize from '@react-hook/google-optimize'
+import { observer } from 'mobx-react'
+import { FC, useEffect } from 'react'
+
+import { Analytics } from '@/common/context/Analytics/components/Analytics'
+import { useCourseGroup } from '@/common/hooks/useCourseGroup'
+import { collectLogEvent } from '@/services/logging'
+import { courseCartStore } from '@/store'
+
+const RecommendationItem = styled(Link)`
+  color: #211091;
+  margin-left: 1em;
+`
+
+const RECOMMENDATION_QUERY = gql`
```

Comment:
the query should be placed in `@/services/apollo/query`

Reply:
ca4f74b